djaoapp/templates/notification/card_updated.eml

```diff
@@ -1,18 +1,18 @@
 {% extends "notification/base.eml" %}
 
 {% block subject %}
-Credit card updated.
+{% trans %}Credit card updated.{% endtrans %}
 {% endblock %}
 
 {% block html_content %}
-<h3>Credit card updated</h3>
+<h3>{% trans %}Credit card updated{% endtrans %}</h3>
 <p>
-The {{organization.printable_name}} credit card has been successfully updated.
+{% trans name=organization.printable_name %}The {{name}} credit card has been successfully updated.{% endtrans %}
```

Comment:
I like this pattern. Could it be applied to some of the places where translation units are broken up in two because we have a link in the middle?